aiohttp/web_exceptions.py

```diff
@@ -420,18 +420,21 @@ class HTTPUnavailableForLegalReasons(HTTPClientError):
 
     def __init__(
         self,
-        link: StrOrURL,
+        link: Optional[StrOrURL] = None,
```

Comment:
Given that the spec says that we _should_ include a URL, do you think we should remove the default? Thus requiring a user to explicitly opt for no link.

Reply:
Umm, you mean the following cases?
```python
# valid construction
HTTPUnavailableForLegalReasons("http://example.com/legal")
HTTPUnavailableForLegalReasons(link="http://example.com/legal")
HTTPUnavailableForLegalReasons(None)
HTTPUnavailableForLegalReasons(link=None)

# invalid construction
HTTPUnavailableForLegalReasons()
```
In this case I would review other exception constructors to make sure this way of doing things is consistent.

Reply:
Well, if there are similar RFC specs that say 'SHOULD', then yeah, it might be worth doing to others. But, those changes could only go into master, as it would break backwards compatibility. We can manage this one now, because it never had a default before.

Reply:
Sounds good to me then. I'll work the argument behaviour like suggested.
Yes, reviewing the others can be a later PR too indeed given the potential breakage.